README.md

```diff
@@ -1,24 +1,11 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Rails Type
 
-Things you may want to cover:
+This app uses Rails 5
 
-* Ruby version
+## What Does It Do?
 
-* System dependencies
+This app serves as a database for Special Transit passengers, to be used by ST dispatchers and administrators. It allows for sorting, filtering, and searching of passengers in the database. We’ve implemented a color code system so ST dispatchers can easily see which passengers are expired, about to expire, deactivated, or lack a needed doctor’s note. This app also has differing admin and non-admin views. A chron job runs every morning at 4:30am to deactivate expired passengers.
```

Comment:
As a general note, I might describe in more user-centric terms what functions it performs at a high level, rather than talking about "sorting, filtering, and searching".